### PR TITLE
Correct validation.rst

### PR DESCRIPTION
**Pull request by LouisHernandez17**
The logit function is actually never used, the correct formulas are these ones, after a check on the original paper and one experimental one.

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to **CausalML**?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ x] I have read the [CONTRIBUTING](https://github.com/uber/causalml/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/uber/causalml)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc. This PR template is adopted from [appium](https://github.com/appium/appium).

### CODE REVIEW

File: docs/validation.rst [hunk 1]
```diff
@@ -79,7 +82,7 @@ Mechanism 4
 |
 | The treatment flag is a binomial variable whose d.g.p. is:
 |
-|   :math:`P(W_i = 1 | X_i) = logit(X_{i1} + X_{i2})`
+|   :math:`P(W_i = 1 | X_i) = \frac{1}{1+\exp{-X_{i1}} + np.exp{-X_{i2}}}`
```
Comment (jeongyoonlee) [inline docs/validation.rst:85]: For `X_{i2}`, `np.exp` --> `\exp`.

**Review by jeongyoonlee — CHANGES_REQUESTED** [review]
Thanks for your  contribution, @LouisHernandez17. I just checked the reference (https://arxiv.org/abs/1712.04912), and you're correct. There is one typo in your update. Once it's fixed, we can merge it.

**Comment by paullo0106** [comment]
Just to confirm, do we then need to make correction to `logit` usage in the data simulation code for [simulate_nuisance_and_easy_treatment](https://github.com/uber/causalml/blob/master/causalml/dataset/regression.py#L71) and  [simulate_unrelated_treatment_control](https://github.com/uber/causalml/blob/master/causalml/dataset/regression.py#L170)?


`e = expit(logit(e) - adj)`

**Review by jeongyoonlee — APPROVED** [review]
LGTM. Thanks for your contribution @LouisHernandez17!

**Pull request merged**